datamodel.combodo-approval-extended.xml

```diff
@@ -355,32 +385,35 @@
 			'first_reject' => ApprovalScheme::EXIT_ON_FIRST_REJECT,
 			'first_approve' => ApprovalScheme::EXIT_ON_FIRST_APPROVE,
 		);
-		if ((get_class($oObject) != 'UserRequest'))
-		{
+
+		$aAllowedClasses = MetaModel::GetConfig()->GetModuleSetting('combodo-approval-extended', 'targets', ApprovalConfiguration::DEFAULT_TARGET);
+        if (array_key_exists(get_class($oObject), $aAllowedClasses) === false)		{
```

Comment:
It would make more sense later in the code when this var is named differently..
```suggestion
		$aTargets = MetaModel::GetConfig()->GetModuleSetting('combodo-approval-extended', 'targets', ApprovalConfiguration::DEFAULT_TARGET);
        if (array_key_exists(get_class($oObject), $aTargets) === false)		{
```